### ISSUE TITLE
Biber bibliography fails

### ISSUE THREAD

**Issue by rudvfaden** [LaTeXing/LaTeXing#110]
When I run `pdflatex + biber + pdflatex (2x)` in sublime text i get the following warnings and no bibliography is created
```
W: /home/rudfaden/Dropbox/PhD/information-and-care/Main/root.tex:0 No file file.bbl.
W: /home/rudfaden/Dropbox/PhD/information-and-care/Main/root.tex:20 Citation 'Starfield1992' on page 1 undefined
...
W: /home/rudfaden/Dropbox/PhD/information-and-care/Main/appendix.tex:78 Citation 'Amir2005' on page 15 undefined
W: /home/rudfaden/Dropbox/PhD/information-and-care/Main/root.tex:0 There were undefined references.
W: /home/rudfaden/Dropbox/PhD/information-and-care/Main/root.tex:0 Please (re)run Biber on the file:(biblatex) file(biblatex) and rerun LaTeX afterwards
```

But when I run the

```
pdflatex root
biber root
pdflatex root
pdflatex root
```

in the terminal evertrhing works fine.

**Comment by csch0** [comment]
and what about latexm?

**Comment by rudvfaden** [comment]
Same problem. Works fine i terminal, but not in sublime.

**Comment by csch0** [comment]
Yes but LaTeXing is doing nothing special with it, just enable the log and
you can see the command called. I think its a problem with the system path
like always... didn't changed anything is this area! So was this ever
working on your machine?

**Comment by rudvfaden** [comment]
Okay. I tried agian. It now works with latexmk, but not when i run `xelatex+biber+2X xelatex`.
I put the log code on pastebin http://pastebin.com/zuxEzriv

**Comment by rudvfaden** [comment]
I don't no if this helps at all, but i found out that you have to remove the `.tex` extension when running biber from the terminal.

**Comment by maxheld83** [comment]
I have the same problem; `pdflatex, biber, pdflatex, pdflatex` works fine on the terminal, but not from within ST3 via LaTeXing.
`latexmk` works both in the terminal and within ST3.

**Comment by csch0** [comment]
ok @maxheld83 could you post your sublime text log during the build?

**Comment by csch0** [comment]
OK just tried this again and its working on my system. So looks like some configuration problem on your system.

**Comment by maxheld83** [comment]
I think I have isolated the bug (?) now, @csch0 (sorry for taking so long), it seems to be related to the output directory:

```
"output_directory": true,
 "output_directory_mode": 0,
```

causes multiple

```
[Compile ~/Github/ODCD-experiments/portfolio-vs-insurance.Rnw]

W: /Users/Max/Github/ODCD-experiments/portfolio-vs-insurance.tex:117 Citation 'Lorenz-2014' on page 1 undefined
```

etc.

**all other options**, such as `"output_directory": false` or `"output_directory": true,
    "output_directory_mode": 1,` work just fine.

**Comment by maxheld83** [comment]
My earlier diagnosis (**now deleted comment**) was wrong – the problem does not stem from `"output_directory": false`, but from `"build_arguments": ["-shell-escape"],`, which I guess is a different issue -> #176
